index.js

```diff
@@ -50,6 +50,13 @@ export default {
       RNPusherPushNotifications.unsubscribe(channel, onError, onSuccess);
     }
   },
+  setUserId: (userId, token, onError, onSuccess) => {
+    if (Platform.OS === 'ios') {
+      RNPusherPushNotifications.setUserId(userId, token, onError);
+    } else {
+      RNPusherPushNotifications.setUserId(userId, token, onError);
+    }
+  },
```

Comment:
Both methods here are the same and this can be reduced to:

```js
  setUserId: (userId, token, onError, onSuccess) => {
       RNPusherPushNotifications.setUserId(userId, token, onError);
  }
```